oshi-core/src/main/java/oshi/software/os/FileSystem.java

```diff
@@ -61,4 +61,12 @@ public interface FileSystem {
      * @return The maximum number of file descriptors if available, 0 otherwise.
      */
     long getMaxFileDescriptors();
+
+    /**
+     * The maximum number of open file descriptors per process. This returns the system-wide limit. The actual limit of
```

Comment:
Using both "per-procsess" and "systemwide" may be confusing.  Can you slightly clarify this?  Perhaps the word "default" or is this the "hard" limit?

Reply:
I tried to add some clarification. Hope this fits better.